src/io/procops.c

```diff
@@ -944,6 +949,9 @@ static void spawn_gc_mark(MVMThreadContext *tc, void *data, MVMGCWorklist *workl
 static void spawn_gc_free(MVMThreadContext *tc, MVMObject *t, void *data) {
     if (data) {
         SpawnInfo *si = (SpawnInfo *)data;
+        if (si->prog) {
+            MVM_free_null(si->prog);
```

Comment:
For curiosity's sake, why the `MVM_free_null` here vs just `MVM_free`? I thought since `si` itself is unconditionally freed at the end, setting the fields to null is not needed?